.rubocop.yml

```diff
@@ -0,0 +1,18 @@
+AllCops:
+  Exclude:
+    - 'db/**/*'
+    - 'config/**/*'
+    - 'script/**/*'
+    - 'bin/{rails,rake}'
+  Include:
```

Comment:
I think we want to check not only 'app', but all other directories that contains code: config, lib, spec